Readme.md

```
@@ -44,3 +38,53 @@ LinkedRecords is configured via environment variables. See table below.
+## Confidential Client Mode (Cookies possible)
+
+| FRONTEND_BASE_URL | http://localhost:3001 | The base URL of the frontend. It will be used for the Access-Control-Allow-Origin HTTP header and is also required for the OpenID connect redirection. |
+| AUTH_ISSUER_BASE_URL | https://xxx.us.auth0.com/ | The URL of the OIDC issuer. Can be any OpenID connect comply identity provider (e.g. Auth0, Okta). |
+| AUTH_CLIENT_ID |  | The client id. Can be obtained from the identity provider. |
+| AUTH_CLIENT_SECRET |  | The client secret. Can be obtained from the identity provider. |
+| AUTH_IDP_LOGOUT | true | When set to true the user session will be destroyed in the application AND the within the identity provider. |
+| AUTH_COOKIE_SIGNING_SECRET | xxxx | The secret used to sign cookies. |
+
+## Public Client Mode
+| ALLOW_HTTP_AUTHENTICATION_HEADER | false | Allows public clients to make requests providing an access token via http authentication header. |
+| AUTH_ISSUER_BASE_URL | https://xxx.us.auth0.com/ | The URL of the OIDC issuer. Can be any OpenID connect comply identity provider (e.g. Auth0, Okta). |
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)